assets/js/components/legacy-notifications/cta.js

```diff
@@ -22,66 +22,58 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
-/**
- * WordPress dependencies
- */
-import { Component } from '@wordpress/element';
-
 /**
  * Internal dependencies
  */
 import Link from '../Link';
 
-class CTA extends Component {
-	render() {
-		const {
-			title,
-			description,
-			ctaLink,
-			ctaLabel,
-			error,
-			onClick,
-		} = this.props;
-
-		return (
-			<div
-				className={ classnames(
-					'googlesitekit-cta',
-					{ 'googlesitekit-cta--error': error }
-				) }
+const CTA = ( {
+	title,
+	description,
+	ctaLink,
+	ctaLabel,
+	error,
+	onClick,
+	'aria-label': ariaLabel,
```

Comment:
Maybe better to use `ariaLabel` for the prop name too here? I see that is already the case for `Link`, but it seems a bit unnecessary to me. What do you think @aaemnnosttv?

Reply:
@felixarntz I tried to keep it consistent with what we have in `Link` but happy to change. Let me know.

Reply:
@aaemnnosttv @tofumatt What do you think? Is it a good practice to use `aria-label` as actual prop for the `aria-label` attribute? Or should we use `ariaLabel` as prop name?

Depending on the decision, we might need to update this PR and also adjust this in the `Link` component.

Reply:
We should use `aria-label` IMO as [all `aria-*` attributes are supported by React](https://reactjs.org/docs/accessibility.html#wai-aria). That way we don't have to worry about which form to use, or whether the component passes extra props through as React will warn if you try to set `ariaLabel` on an HTML element in JSX:
![image](https://user-images.githubusercontent.com/1621608/101770529-3d9a8c00-3af1-11eb-8048-94d7640ac9e5.png)

Within the component, I think it's fine to alias it to camel case like we do in `Link`, but between components I think we should always use `aria-label` for consistency.

Reply:
Agreed, we should use `aria-label` as the actual prop name everywhere for consistency. It's what I would expect as a React developer—and it's good to be consistent. I wouldn't expect to use `ariaLabel` as it's an HTML element and I know React has explicit support for `aria-*` props.